lib/meataxe.gi

```diff
@@ -3096,11 +3096,11 @@ local cf,dim,b,den,sub,i,s,q,found,qb;
             Add(sub,TriangulizedMat(b));
             den:=dim;
           else
-            TriangulizeMat(s);
+            s:=TriangulizedMat(s);
```

Comment:
The problem is that `s` can be immutable (in fact, with this PR, it is *always* immutable), and then this function runs into an error. Hence we must make a copy.